out.txt

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
if this is a test file, it seems that the script doesn't quite work as expected

Reply:
Yeah, the changes I made this afternoon weren't quite ready for another review it seems 😅